src/NServiceBus.MessagingBridge/MessageShovel.cs

```diff
@@ -114,25 +126,37 @@ static void TransformRegularMessageReplyToAddress(
         }
         else
         {
-            messageToSend.Headers[Headers.ReplyToAddress] = targetEndpointRegistry.TranslateToTargetAddress(headerValue);
+            bool throwOnError = !(translateReplyToAddressForFailedMessages && IsRetryEditedMessage(messageToSend));
```

Comment:
I'm not sure if this is the best place for this logic.  When reviewing the original PR with @tmasternak we discussed the need for having `TransformRegularMessageReplyToAddress` vs just having conditional logic in `TransformHeaderAddress`, and it was thought that a specific method for regular messages had clearer intent.

Now I'm wondering if we should have a `TransformEditedMessageReplyToAddress` method as well to be consistent, rather than conditional logic here?

Reply:
I think `TransformEditedMessageReplyToAddress` might be overkill but I have adjusted it to remove the logic from that method.